infra/azure.tf

```diff
@@ -145,10 +145,31 @@ CRON
 
 chmod +x /root/daily-reset.sh
 
+cat <<'GET_TARGETS' > /root/get-targets.sh
+#!/usr/bin/env bash
+set -euo pipefail
+
+arg_target="$1"
+
+month=$(date +%m)
+day_of_month=$(date +%d)
+if [[ "$month" -eq 12 && "$day_of_month" -gt 22 ]]; then
+  # We treat the days after December 22nd as weekend days.
```

Comment:
the 2024 DA Winter Break starts on (Monday) Dec 23 so we shouldn't need to change it this year